sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -979,18 +979,18 @@ private void onSampleChanged(SampleDto existingSample, Sample newSample, boolean
 
 	private void handleAssotiatedObjectChanges(Sample newSample, boolean syncShares) {
 		if (newSample.getAssociatedCase() != null) {
-			caseFacade.onCaseChanged(CaseFacadeEjbLocal.toDto(newSample.getAssociatedCase()), newSample.getAssociatedCase(), syncShares);
+			caseFacade.onCaseChanged(CaseFacadeEjb.toDto(newSample.getAssociatedCase()), newSample.getAssociatedCase(), syncShares);
```

Comment:
Afaik there's a reason we're using the local bean here; @MartinWahnschaffe can you share some insight?

Reply:
I think the warning originated from the usage of a static function on the extended class.

Reply:
What you have in mind is that when using an instance of an EJB in the backend, you have to use the "local" instance.

The toDto method is a static method though, so there is no difference between using the static method via the class name of the ejb or any inherited class. As Jonas points out, the system will give a warning that says we should use the class that acutally defines the static method.

TL;DR: The refactoring is good the way it is.

Reply:
:+1: